cmd/policy_enable.go

```diff
@@ -25,35 +25,27 @@ import (
 
 type policyEnableArgs struct {
 	policyGroup string
-	latest      bool
 }
 
 func newPolicyEnableCmd() *cobra.Command {
 	args := policyEnableArgs{}
 
 	var cmd = &cobra.Command{
-		Use:   "enable <org-name>/<policy-pack-name> [version]",
-		Args:  cmdutil.RangeArgs(1, 2),
+		Use:   "enable <org-name>/<policy-pack-name> <latest|version>",
+		Args:  cmdutil.ExactArgs(2),
 		Short: "Enable a Policy Pack for a Pulumi organization",
-		Long:  "Enable a Policy Pack for a Pulumi organization. Version or latest flag must be specified.",
+		Long: "Enable a Policy Pack for a Pulumi organization. " +
+			"Can specify latest to enable the latest version of the Policy Pack or a specific version number.",
 		Run: cmdutil.RunFunc(func(cmd *cobra.Command, cliArgs []string) error {
 			// Obtain current PolicyPack, tied to the Pulumi service backend.
 			policyPack, err := requirePolicyPack(cliArgs[0])
 			if err != nil {
 				return err
 			}
 
-			// Make sure that a version or latest is specified. Having both or neither
-			// specified would make this an ambiguous request.
-			if len(cliArgs) < 2 && !args.latest {
-				return errors.New("must specify a version or the --latest flag")
-			} else if len(cliArgs) == 2 && args.latest {
-				return errors.New("cannot specify both a version and the --latest flag")
-			}
-
 			// Parse version if it's specified.
 			var version *int
-			if len(cliArgs) > 1 {
+			if cliArgs[1] != "latest" {
```

Comment:
Nit: Did you want to do something like `const latestKeyword = "latest"`, like with the other command?